Factory/ProducerFactory.php

```diff
@@ -20,12 +20,17 @@ public function get(string $producerClass, array $producerData): ProducerManager
     {
         $producerManager = new ProducerManager();
         $this->getReadyConfiguration($producerData['configuration']);
+
+        $this->configuration->setDrMsgCb([$producerManager, 'produceResponse']);
+        $this->configuration->setErrorCb([$producerManager, 'produceError']);
+
         $producer = new $producerClass($this->configuration);
 
         $producerManager->setProducer($producer);
         $producerManager
             ->addBrokers(implode(',', $producerData['brokers']))
             ->setLogLevel($producerData['log_level'])
+            ->setEventsPoolTimeout($producerData['events_poll_timeout'])
```

Comment:
you call it `setEventsPoolTimeout` but your param is called `events_poll_timeout` (pool > poll)

could you harmonize it?